query/aql_compiler.go

```diff
@@ -967,6 +975,42 @@ func (qc *AQLQueryContext) Rewrite(expression expr.Expr) expr.Expr {
 			}
 
 			e.ExprType = expr.Boolean
+		case expr.PointInPolygonCallName:
```

Comment:
can we still use geo_intersect call for this case

Reply:
it's going to be tricking to reuse the name. as in qc.rewrite, we validate arg types base on call name, but not where the expr came from.